includes/compat/class-wc-stripe-subs-compat.php

```diff
@@ -506,4 +510,47 @@ public function maybe_render_subscription_payment_method( $payment_method_to_dis
 
 		return $payment_method_to_display;
 	}
+
+	/**
+	 * Loads a list of previous charges for subscriptions.
+	 *
+	 * @param  object $prepared_source All available data about the payment instrucment.
+	 * @param  object $order           The order that is being created for the renewal.
+	 * @return array                   A list of charge IDs.
+	 */
+	public function load_previous_charges( $prepared_source, $order ) {
```

Comment:
All this is speculation, right? We don't actually know what data Stripe will require. I think it would be better to either comment out this function, or split this PR into 2: one for fixing what was broken, and another one to fetch the previous charges.

Reply:
That is a very good point. I wanted to have something prepared and in the process it turned out that I need to re-add the needed metadata to charges.

To separate the PRs I created [a new branch](https://github.com/woocommerce/woocommerce-gateway-stripe/compare/sca/add-charge-metadata), but until https://github.com/woocommerce/woocommerce-gateway-stripe/pull/826 gets reviewed/merged it doesn't make sense to resolve conflicts that will re-appear.

Let's put this PR on hold now.